test/utilities/FormManager.test.js

```diff
@@ -0,0 +1,38 @@
+/* global describe, it, expect */
+/* eslint-disable no-magic-numbers, sort-keys */
```

Comment:
We don't have eslint in this repository, right? Do you think we should add it (seeing `import {Tester}` below made me notice the lack of linting)?

Reply:
We do not! Must be a copy-paste problem. Also, I believe Jest now has native TS support so I'd like to attempt that and gain tslint, but registry blockers are obviously more important.